userguide/system.rst

```diff
@@ -2288,21 +2288,52 @@ Updating an HA System
 
 If the %brand% array has been configured for High Availability
 (HA), the update process must be started on the active |ctrlr-term|.
-Once the update is complete, the standby |ctrlr-term| will automatically
-reboot. Wait for it to come back up by monitoring the remote console or
-the |web-ui| of the standby |ctrlr-term|.
+The update process is to download and synchronize the update files
+between both |ctrlrs-term|, update and reboot the standby |ctrlr-term|,
+manually activate the standby |ctrlr-term|, then install the update on
+the previously active |ctrlr-term|.
 
-After the standby |ctrlr-term| has finished booting, it is important to
-perform a failover by rebooting the current active |ctrlr-term|. This
-action tells the standby |ctrlr-term| to import the current
-configuration and restart services.
+Starting an HA update shows a confirmation dialog that also explains
+this update process. To start the HA update, click :guilabel:`CONTINUE`.
+A dialog shows the progress of downloading, syncing, and installing the
+update.
 
-Once the previously active |ctrlr-term| comes back up as a standby
-|ctrlr-term|, use
-:menuselection:`System --> Update`
-to apply the update on the current active |ctrlr-term| (which was
-previously the passive |ctrlr-term|). Once complete, the now standby
-|ctrlr-term| will reboot a second time.
+.. note:: All other clients logged in to the system will see a dialog
+   explaining that an update is in progress and that the system will
+   restart when the update is complete.
+
+
+The standby |ctrlr-term| reboots after installing the update. This can
+take some time. When it is back online, the |web-ui| prompts to continue
+the update process by initiating a :ref:`Failover`.
+
+.. figure:: images/truenas/system-update-ha-failover.png
+
+
+Continue the update by going to the
+:menuselection:`Dashboard`
+and clicking :guilabel:`INITIATE FAILOVER`. Wait for the :ref:`failover`
+process to finish and log back in to the |web-ui|. If the log in screen is
+not shown, enter the IP address of the previous standby |ctrlr-term|
```

Comment:
Should this be the VIP?

Reply:
Not necessarily. In testing, the user can use the VIP or the direct IP address of the controller. If you're using the VIP, the webui automatically takes you to the log in screen. If you're connected directly to the controller that went down, the screen shows the IP of the other controller as an option for connecting to the web ui. That is why it is written as a conditional sentence.